bundle/edu.gemini.itc.web/src/main/resources/servlets/web/ITCacqCam.html

```diff
@@ -254,20 +248,22 @@
   <p><font color="#FF0000"><b>Instrument (Acquisition Camera) and telescope configuration</b></font></p>
   <table border="0" width="100%" background="https://www.gemini.edu/sciops/instruments/itc/mez-green-bckgrnd.gif" cellpadding="6" cellspacing="0">
     <tr>
-      <td colspan="4"><b>Instrument optical properties:</b><i><font size="-1">(</font><a href="#" onclick="window.open('http://www.gemini.edu/?q=node/10274#optical','mywindow','width=1000,height=700,toolbar=no,location=no,directories=no,sta tus=no,menubar=no,scrollbars=yes,copyhistory=no,resizable=no'); return false"><font size="-1">more info</font></a><font size="-1">)</font></i></td>
+      <td colspan="1"><b>Instrument optical properties:</b><i><font size="-1">(</font><a href="#" onclick="window.open('http://www.gemini.edu/?q=node/10274#optical','mywindow','width=1000,height=700,toolbar=no,location=no,directories=no,sta tus=no,menubar=no,scrollbars=yes,copyhistory=no,resizable=no'); return false"><font size="-1">more info</font></a><font size="-1">)</font></i></td>
```

Comment:
This seems like it makes more sense. I wonder why it was set to 4 beforehand?

Reply:
It was probably copied and pasted from some other instrument which used colspan=4.